templates/github/.ci/scripts/publish_plugin_pypi.sh.j2

```diff
@@ -7,6 +7,14 @@ cd "$(dirname "$(realpath -e "$0")")"/../..
 
 set -euv
 
+export VERSION=$(http pulp/pulp/api/v3/status/ | jq --arg plugin {{ plugin_app_label }} -r '.versions[] | select(.component == $plugin) | .version')
+export response=$(curl --write-out %{http_code} --silent --output /dev/null https://pypi.org/project/pulp-npm-client/$VERSION/)
+if [ "$response" == "200" ];
+then
+  echo "{{ plugin_name }} $VERSION has already been released. Skipping."
+  exit
+fi
```

Comment:
This code is basically copied over from the other release scripts such as publish_client_pypi.sh.

Reply:
why draft? It seems ready to me